### ISSUE TITLE
Exception thrown due to deprecated method - Rider 2024.1

### ISSUE THREAD

**Issue by gwynnPalmer** [didalgolab/chatgpt-intellij-plugin#20]
Exception thrown - doesn't appear to impact functionality.
Just in case you were not already aware 

```
com.intellij.diagnostic.PluginException: `ActionUpdateThread.OLD_EDT` is deprecated and going to be removed soon. 'com.didalgo.intellij.chatgpt.ui.action.editor.ExplainAction' must override `getActionUpdateThread` and chose EDT or BGT. See ActionUpdateThread javadoc. [Plugin: com.didalgo.chatgpt]
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:23)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:90)
	at com.intellij.diagnostic.PluginException.reportDeprecatedUsage(PluginException.java:125)
	at com.intellij.openapi.actionSystem.ActionUpdateThreadAware.getActionUpdateThread(ActionUpdateThreadAware.java:21)
	at com.intellij.openapi.actionSystem.AnAction.getActionUpdateThread(AnAction.java:199)
```

Great plugin by the way. Cheers.

**Comment by didalgolab** [comment]
Thanks, this will be fixed on the next release.

**Comment by didalgolab** [comment]
Fixed in v1.0.0 by 469cd937
